test/sampler.jl

```diff
@@ -85,7 +85,7 @@ end
                     # For other adapatation methods that are able to adpat the step size, we use `find_good_stepsize`.
                     τ_used = adaptorsym == :MassMatrixAdaptorOnly ? τ : reconstruct(τ, integrator=reconstruct(lf, ϵ=find_good_stepsize(h, θ_init)))
                     samples, stats = sample(h, τ_used , θ_init, n_samples, adaptor, n_adapts; verbose=false, progress=PROGRESS)
-                    @test mean(samples) ≈ zeros(D) atol=RNDATOL
+                    @test mean(samples[(n_adapts+1):end]) ≈ zeros(D) atol=RNDATOL
```

Comment:
Maybe use `drop_warmup=true` in the `sample` call above.

Reply:
yes, but the effect should be similar to what we currently do.